rasa/core/policies/fallback.py

```diff
@@ -112,6 +112,9 @@ def predict_action_probabilities(self,
             # predict fallback action with confidence `core_threshold`
             # if this is the highest confidence in the ensemble,
             # the fallback action will be executed.
+            logger.debug("Predict fallback action with confidence "
+                         "'core_threshold' ({}). "
+                         "".format(self.core_threshold))
```

Comment:
I'm thinking something like "NLU confidence threshold met, confidence of fallback action set to core threshold ({})." might make it more clear that this part doesn't actually predict the fallback action, and might reflect the docs well -- 

> The FallbackPolicy invokes a fallback action if the intent recognition has a confidence below `nlu_threshold` or if none of the dialogue policies predict an action with confidence higher than `core_threshold`.

https://rasa.com/docs/core/master/policies/#fallback-policy

Reply:
and I definitely like this clean solution a lot better than messing stuff up 😄